build/charts/antrea-windows/values.yaml

```diff
@@ -0,0 +1,10 @@
+# -- Container image used for Antrea Windows Nodes
+image:
+  repository: "antrea/antrea-windows"
+  tag: "latest"
+
+# -- Enable release mode
+release: false
```

Comment:
sorry if I missed it, but I didn't see this used in any template file?

Reply:
Ah thats from a previous commit that I reverted. It is not used anywhere.